vendor/github.com/mongodb/grip/vendor/github.com/shirou/gopsutil/process/process_windows.go

```diff
@@ -278,22 +278,22 @@ func (p *Process) MemoryInfoEx() (*MemoryInfoExStat, error) {
 }
 
 func (p *Process) Children() ([]*Process, error) {
-	procs, err := processes()
+	var dst []Win32_Process
+	query := wmi.CreateQuery(&dst, fmt.Sprintf("Where ParentProcessId = %d", p.Pid))
+	err := wmi.Query(query, &dst)
 	if err != nil {
 		return nil, err
 	}
-	out := []*Process{}
 
-	for _, proc := range procs {
-		parent, err := proc.Parent()
+	out := []*Process{}
+	for _, proc := range dst {
+		p, err := NewProcess(int32(proc.ProcessID))
 		if err != nil {
 			continue
 		}
-
-		if parent.Pid == p.Pid {
-			out = append(out, proc)
-		}
```

Comment:
can we write a test to make sure that we don't include parent pids in child pid arrays?